src/LiteCore/test/LiteCore.Tests.Shared/ScopeCollectionTest.cs

```diff
@@ -0,0 +1,667 @@
+                ((long)doc->revID.buf).Should().Be(0, "because the doc has no revision ID yet");
+                ((long)doc->selectedRev.revID.buf).Should().Be(0, "because the doc has no revision ID yet");
+                Native.c4doc_release(doc);
+
+                LiteCoreBridge.Check(err => Native.c4db_beginTransaction(Db, err));
+                try {
+                    var tmp = RevID;
+                    var rq = new C4DocPutRequest {
+                        existingRevision = true,
+                        docID = DocID,
+                        history = &tmp,
+                        historyCount = 1,
+                        body = FleeceBody,
+                        save = true
+                    };
+
+                    doc = (C4Document*)LiteCoreBridge.Check(err => {
+                        var localRq = rq;
+                        return Native.c4coll_putDoc(DefaultColl, &localRq, null, err);
+                    });
+                    doc->revID.Equals(RevID).Should().BeTrue("because the doc should have the stored revID");
+                    doc->selectedRev.revID.Equals(RevID).Should().BeTrue("because the doc should have the stored revID");
+                    doc->selectedRev.flags.Should().Be(C4RevisionFlags.Leaf, "because this is a leaf revision");
+                    NativeRaw.c4doc_getRevisionBody(doc).Equals(FleeceBody).Should().BeTrue("because the body should be stored correctly");
+                    Native.c4doc_release(doc);
+                } finally {
+                    LiteCoreBridge.Check(err => Native.c4db_endTransaction(Db, true, err));
+                }
+
+                // Reload the doc:
+                doc = (C4Document*)LiteCoreBridge.Check(err => NativeRaw.c4coll_getDoc(DefaultColl, DocID, true, C4DocContentLevel.DocGetCurrentRev, err));
+                doc->flags.Should().Be(C4DocumentFlags.DocExists, "because this is an existing document");
+                doc->docID.Equals(DocID).Should().BeTrue("because the doc should have the stored doc ID");
+                doc->revID.Equals(RevID).Should().BeTrue("because the doc should have the stored rev ID");
+                doc->selectedRev.revID.Equals(RevID).Should().BeTrue("because the doc should have the stored rev ID");
+                doc->selectedRev.sequence.Should().Be(1, "because it is the first stored document");
+                NativeRaw.c4doc_getRevisionBody(doc).Equals(FleeceBody).Should().BeTrue("because the doc should have the stored body");
+                Native.c4doc_release(doc);
+
+                // Get the doc by its sequence
+                /** Gets a document from the collection given its sequence number.
+                @note  You must call `c4doc_release()` when finished with the document.  */
+                doc = (C4Document*)LiteCoreBridge.Check(err => Native.c4coll_getDocBySequence(DefaultColl, 1, err));
```

Comment:
Looks like c4coll_getDocBySequence is not exposed for c# to use. I am getting 
System.EntryPointNotFoundException : Unable to find an entry point named 'c4coll_getDocBySequence' in DLL 'LiteCore'.

Reply:
Looks like c4coll_getDocBySequence is not needed. I will take this out from the binding test.

Reply:
This was an oversight but I don't think this API is used for anything anyway (at least not by CBL)

Reply:
Removed